SourceKitStressTester/Tests/StressTesterToolTests/StressTesterToolTests.swift

```diff
@@ -81,6 +122,29 @@ class StressTesterToolTests: XCTestCase {
     XCTAssertFalse(responses.allSatisfy { $0.results.isEmpty }, "responses have results")
   }
 
+  func testModuleRequest() {
```

Comment:
I'm not sure this test really adds much at all... I did have it testing the action itself but that obviously won't work until the option gets into the latest toolchain

Reply:
I think this is fine for now. You can make it do the actual action once it's available in a toolchain. It might be good to do something like the wrapper tests where it uses `makeScript` to set up a fake swiftc that records the arguments it was invoked with and returns a fixed status code so you can test you're invoking swiftc with the arguments you expect when generating the module.

Reply:
Ah yeah, that's not a bad idea - will do.